api/gateway_test.go

```diff
@@ -134,3 +142,4 @@ func TestBlockBootstrap(t *testing.T) {
 		time.Sleep(time.Millisecond)
 	}
 }
+*/
```

Comment:
Is this code commented out because something is incomplete, or is it no longer necessary?